providers/fetcher/githubProcessor.js

```diff
@@ -701,8 +706,17 @@ class GitHubProcessor {
   }
 
   PushEvent(request) {
-    let [document] = this._addEventBasics(request);
-    // TODO figure out what to do with the commits
+    let [document, repo, payload] = this._addEventBasics(request);
+    const qualifier = `urn:repo:${repo}:PushEvent:${document.id}`;
```

Comment:
this is already computed in _addEventBasics as document._metadata.links.self.href.  Would be good to use that for consistency